WordPress/Classes/Models/BasePost.h

```diff
@@ -32,9 +39,40 @@ typedef enum {
 @property (nonatomic) AbstractPostRemoteStatus remoteStatus;
 @property (nonatomic, strong) NSNumber * post_thumbnail;
 
-@property (nonatomic, assign) BOOL isFeaturedImageChanged;
+// Helpers
+@property (nonatomic, strong) NSString *pathForDisplayImage; // Cached path of an image from the post to use for display purposes. Not part of the post's canoncial data.
+@property (nonatomic, assign) BOOL isFeaturedImageChanged; // BOOL flag if the feature image was changed.
 
-- (NSArray *)availableStatuses;
+/**
+ Returns the localized title for the specified status.  Status should be
+ one of the `PostStatus...` constants.  If a matching title is not found
+ the status is returned.
+
+ @param string The post status value
+ @return The localized title for the specified status, or the status if a title was not found.
+ */
++ (NSString *)titleForStatus:(NSString *)status;
+
+/**
+ Transforms the specified string to plain text.  HTML markup is removed and HTML entities are decoded.
+
+ @param string The string to transform.
+ @return The transformed string.
+ */
++ (NSString *)makePlainText:(NSString *)string;
```

Comment:
This method is an odd one to have here since it seems to be very generic and not strictly related to posts.  What about moving this to `NSString+Helpers`?